docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -140,6 +140,12 @@ discovery_service:
        "env": "prod" # Match EC2 instances where tag:env=prod
 ```
 
+- Edit the `teleport.auth_servers` key to match your Auth or Proxy server URI
```

Comment:
```suggestion
Edit the `teleport.auth_servers` key to match your Auth Service or Proxy Service's URI
```

Reply:
I'm 👍 on the changes at the end of the sentence, but why remove the UL marker?

Reply:
That was a mistake—I'll fix the suggestion